README.md

```diff
@@ -217,8 +217,8 @@ checked):
 Implementations are expected to not error when any of the following context
 files are used in a verifiable credential or a verifiable presentation:
 
-- [VC 2.0 Context - https://www.w3.org/ns/credentials/v2](https://www.w3.org/ns/credentials/v2)
-- [VC Examples Context - https://www.w3.org/ns/credentials/examples/v2](https://www.w3.org/ns/credentials/examples/v2)
+- [Vc 2.0 Context - https://www.w3.org/ns/credentials/v2](https://www.w3.org/ns/credentials/v2)
+- [Vc Examples Context - https://www.w3.org/ns/credentials/examples/v2](https://www.w3.org/ns/credentials/examples/v2)
```

Comment:
Well...this takes the code style into the wrong place. 😃 These are for people, who typically abbreviate Verifiable Credentials as VC...and would read `Vc` as a typo. So, the README needs refixing.

Reply:
This should make the corrections asked for: https://github.com/w3c/vc-data-model-2.0-test-suite/commit/eda2a3b473cab756fa890aed271cab870fdf55db